showcase/config/webpack-custom-config.dev.json

```diff
@@ -1,4 +1,5 @@
 {
   "cspConnectSrc": "http://localhost:5000 http://localhost:5001 http://localhost:5002 http://localhost:4000 https://nationalbankbelgium.github.io",
-  "cspFormAction": "http://localhost:5000/myAwesomeUpload"
+  "cspFormAction": "http://localhost:5000/myAwesomeUpload",
+  "cspFontSrc": "https://fonts.gstatic.com"
```

Comment:
For the moment we won't add a custom font in the Showcase, we will just mention this feature in the docs ;)